pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -232,8 +232,8 @@ func (i *KubeAPIServerInformers) GetInternalKubernetesInformers() kinternalinfor
 func (i *KubeAPIServerInformers) GetKubernetesInformers() kexternalinformers.SharedInformerFactory {
 	return i.KubernetesInformers
 }
-func (i *KubeAPIServerInformers) GetInternalOpenshiftImageInformers() imageinformer.SharedInformerFactory {
-	return i.InternalOpenshiftImageInformers
+func (i *KubeAPIServerInformers) GetOpenshiftImageInformers() imagev1informer.SharedInformerFactory {
```

Comment:
wait, if this renamed worked, who is using this thing?

Reply:
I am unable to find any users of it prior to this change